### PR TITLE
[4533] Update `ReactionsPicker` docs

### LINKED ISSUE #4533: ReactionsPicker

### PR DESCRIPTION
**Pull request by MarinTolic**
### 🎯 Goal

closes #4533

### 🧪 Testing

Launch docusaurus and evaluate the new changes.

#### General
- [x] Assigned a person / code owner group (required)
- [x] Thread with the PR link started in a respective Slack channel (#android-chat-core or #android-chat-ui) (required)
- [x] PR targets the `develop` branch
- [x] PR is linked to the GitHub issue it resolves

#### Code & documentation
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [x] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

### 🎉 GIF

gif

### CODE REVIEW

File: docusaurus/android_versioned_docs/version-5.x.x/04-compose/05-message-components/08-reactions-picker.mdx [hunk 1]
```diff
@@ -36,18 +36,13 @@ The code above will render the following UI:
 
 ![Default ReactionsPicker Component](../../assets/compose_default_reactions_picker_component.png)
 
-:::note
-`MessagesScreen` will give you the ability to display `ReactionsPicker` only if you have defined more than 5 reactions inside of [`ChatTheme`](../03-general-customization/01-chat-theme.mdx).
```
Comment (MarinTolic) [inline docusaurus/android_versioned_docs/version-5.x.x/04-compose/05-message-components/08-reactions-picker.mdx:40]: I think this was a mistake originally.

Tried launching it with only a single reaction and it worked fine.

File: docusaurus/docs/Android/04-compose/05-message-components/08-reactions-picker.mdx [hunk 2]
```diff
@@ -87,7 +84,7 @@ Next, let's see how to customize `ReactionsPicker`.
 @Composable
```
Comment (bychkovdmitry) [inline docusaurus/docs/Android/04-compose/05-message-components/08-reactions-picker.mdx:84]: @ExperimentalFoundationApi

File: docusaurus/docs/Android/04-compose/05-message-components/08-reactions-picker.mdx [hunk 3]
```diff
@@ -87,7 +84,7 @@ Next, let's see how to customize `ReactionsPicker`.
...
 public fun ReactionsPicker(
     ..., // State,
-    reactionTypes: Map<String, Int> = ChatTheme.reactionTypes,
+    reactionTypes: Map<String, ReactionIcon> = ChatTheme.reactionIconFactory.createReactionIcons(),
     ... // Actions and content
 )
 ```
```
Comment (bychkovdmitry) [inline docusaurus/docs/Android/04-compose/05-message-components/08-reactions-picker.mdx:90]: `By default it uses reactionIconFactory inside of...`

`You can customize these either by overriding ChatTheme.reactionIconFactory`

File: docusaurus/docs/Android/04-compose/05-message-components/08-reactions-picker.mdx [hunk 4]
```diff
@@ -87,7 +84,7 @@ Next, let's see how to customize `ReactionsPicker`.
 @Composable
 public fun ReactionsPicker(
     ..., // State,
-    reactionTypes: Map<String, Int> = ChatTheme.reactionTypes,
+    reactionTypes: Map<String, ReactionIcon> = ChatTheme.reactionIconFactory.createReactionIcons(),
```
Comment (bychkovdmitry) [inline docusaurus/docs/Android/04-compose/05-message-components/08-reactions-picker.mdx:87]: Line 141: "Empty by default."

Comment (bychkovdmitry) [inline docusaurus/docs/Android/04-compose/05-message-components/08-reactions-picker.mdx:87]: Line 91: "Allows you to customize"

**Review by bychkovdmitry — APPROVED** [review]
Nice job! Couple of comments related to reaction factory (`ChatTheme.reactionIconFactory`).

File: docusaurus/docs/Android/04-compose/05-message-components/08-reactions-picker.mdx (hunk 2 above)
Comment (MarinTolic) [reply docusaurus/docs/Android/04-compose/05-message-components/08-reactions-picker.mdx:84]: Removed

File: docusaurus/docs/Android/04-compose/05-message-components/08-reactions-picker.mdx (hunk 4 above)
Comment (MarinTolic) [reply docusaurus/docs/Android/04-compose/05-message-components/08-reactions-picker.mdx:87]: Replaced the comma with a full stop

**Pull request merged**